tiingo/test/adapter.test.ts

```diff
@@ -0,0 +1,83 @@
+import { assert } from 'chai'
+import { Requester } from '@chainlink/external-adapter'
+import { assertSuccess, assertError } from '@chainlink/adapter-test-helpers'
+import { AdapterRequest } from '@chainlink/types'
+import { makeExecute } from '../src/adapter'
+
+describe('execute', () => {
+  process.env.API_KEY = process.env.API_KEY ?? 'aa8ea0dd8555e13832a637932449d009f2ca441f'
```

Comment:
```suggestion
  process.env.API_KEY = process.env.API_KEY ?? 'test_API_key'
```
You'll want to keep API keys out of the code base.

We still need to connect this bit to a secrets store for local integration testing.

Reply:
Ahhh that makes more sense, was a bit confused when I thought you meant to include a key